### PR TITLE
added markdownlint maker for markdown

### PR DESCRIPTION
**Pull request by sbdchd**

### CODE REVIEW

**Comment by blueyed** [comment]
Thanks.

Does it make sense to use both 'proselint' and 'markdownlint' (by default) when they are installed together, or should one of them get preferred?

**Comment by sbdchd** [comment]
@blueyed I think it makes sense to only have one markdown linter enabled by default, `mdl` or `markdown lint`. 

[Proselint](http://proselint.com) serves a different purpose in that it doesn't lint markdown syntax, instead it lints the contents of a file, the proses.

So I think it is best to have `proselint` enabled along side a markdown linter.

File: README.md [hunk 1]
```diff
@@ -170,7 +170,8 @@ Standard ML:
 - smlnj
 
 Markdown:
-- mdl
+- [markdownlint](https://github.com/igorshubovych/markdownlint-cli)
+- [mdl](https://github.com/mivok/markdownlint)
```
Comment (blueyed) [inline README.md]: Shouldn't proselint be added here then, too?

**Comment by sbdchd** [comment]
updated markdown portion of readme to include proselint

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**